docker-compose-sunbird/.env

```diff
@@ -10,9 +10,10 @@ VAULT_BASE_URL=http://vault:8200/v1
 VAULT_ROOT_PATH=http://vault:8200/v1/kv
 VAULT_TIMEOUT=5000
 VAULT_PROXY=false
-SIGNING_ALGORITHM=Ed25519
+SIGNING_ALGORITHM=Ed25519Signature2020
 JWKS_URI=
 ENABLE_AUTH=false
+WEB_DID_BASE_URL=https://challabeehyv.github.io/DID-Resolve
```

Comment:
In the request from plugin to Sunbird C the issuer Id is used for  credential issuance,WEB_DID_BASE_URL  is used while creating an issuer DID, we have added steps to make it resolvable so that users can verify the generated credential.

Reply:
what if there are 2 different DIDs and I want to generate DID from 2 different git repo ? What should be configured here ?

Reply:
Currently sunbird only supports one git repo, but we can host multiple DIDs in a single repo. Added the steps for the same.

Reply:
As of now, this is fine. But this would be a problem when we want to support multi tenancy. Also I would like to understand why do have such a restriction ?